test/unit/specs/App.spec.js

```diff
@@ -59,11 +57,11 @@ describe(`App without analytics`, () => {
     })
     let Node = require(`renderer/connectors/node.js`)
     require(`renderer/main.js`)
-    expect(Node).toHaveBeenCalledWith(
+    expect(Node.mock.calls[0].slice(1)).toEqual([
```

Comment:
What was the problem here?

Reply:
@NodeGuy ^^

Reply:
I added an initial argument `axios` to that function and `toHaveBeenCalledWith` was having trouble comparing functions so I opted to bypass it.